src/adagrams.js

```diff
@@ -1,15 +1,158 @@
+const letterPool = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
+const scoresChart = {
+  A: 1,
+  C: 3,
+  B: 3,
+  E: 1,
+  D: 2,
+  G: 2,
+  F: 4,
+  I: 1,
+  H: 4,
+  K: 5,
+  J: 8,
+  M: 3,
+  L: 1,
+  O: 1,
+  N: 1,
+  Q: 10,
+  P: 3,
+  S: 1,
+  R: 1,
+  U: 1,
+  T: 1,
+  W: 4,
+  V: 4,
+  Y: 4,
+  X: 8,
+  Z: 10,
+};
+
 export const drawLetters = () => {
   // Implement this method for wave 1
+  let tenLetters = [];
+  const letterPoolTwo = { ...letterPool };
```

Comment:
While this does pass the tests, I think simply capturing the keys and checking against their count will not give us an accurate probability distribution.

Each of the 26 characters still have the same probability, which _technically_ they shouldn't. Something to consider is making one big array that appends a letter for every count, ie `["A","A","A","B","B"....]`. That would make sure that when an index is randomly selected, not every letter has the same probability!